ansible_collections/arista/avd/examples/cv-pathfinder/intended/structured_configs/pf1.yml

```diff
@@ -654,6 +654,7 @@ router_path_selection:
     flow_assignment: lan
   load_balance_policies:
   - name: LB-BLUE-POLICY-VIDEO
+    lowest_hop_count: false
```

Comment:
@MaheshGSLAB / @Vibhu-gslab we accept this change

Reply:
I think we should remove the default value from the schema. Clearly we did not respect it before.